jubatus/server/framework/mixer/linear_mixer.cpp

```diff
@@ -476,6 +476,11 @@ void linear_mixer::mix() {
                 diff_result.error[i].host(), diff_result.error[i].port()));
         }
 
+        if (diff == 0) {  // all get_diffs fail
```

Comment:
Why not `if (!diff)`?
cf. https://github.com/gwtnb/jubatus/blob/prevent-segv-in-mix/jubatus/server/framework/mixer/linear_mixer.cpp#L469